stats.py

```diff
@@ -57,8 +57,12 @@ def __sub__(self, stats):
 
     def print_stats(self):
         """pretty print the stats"""
-        return print("\n".join([f"{k}: {v}" for k, v in self.__dict__.items()]))
+        return print("\n".join([f"{k}: {v}" for k, v in self._dict.items()]))
 
     def get(self, attribute: str, default: float):
         """Similar to a dict get with default value"""
         return self._dict.get(attribute, default)
```

Comment:
Is this better than just being able to do self.orig_base_stats.ability_power ?

Reply:
The magic method __getattr__ has be rewritten so that when you do stats.ability_power it works just like a class attribute. The get method has a default value, and behaves like dict.get().

Reply:
Ok i see